.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -21,9 +21,9 @@ body:
           required: true
         - label: Check that there isn't [already an issue](https://github.com/uab-cgds-worthey/rosalution/issues/) that reports the same bug to avoid creating a duplicate.
           required: true
-        - label: Follow our [Code of Conduct](../../CODE_OF_CONDUCT.md)
+        - label: Follow our [Code of Conduct](CODE_OF_CONDUCT.md)
```

Comment:
I tested out the link outs and I was redirected to the following URL: https://github.com/uab-cgds-worthey/CODE_OF_CONDUCT.md .  I took a step back and looked at some of the examples we derived the templates from and noted that fully qualified URLS were linked to in the templates https://raw.githubusercontent.com/vitejs/vite/main/.github/PULL_REQUEST_TEMPLATE.md .  Lets go ahead and use these for the templates to avoid confusion, since we know that these work.

Reply:
made this suggested change

Reply:
All of the links modified in all of the files in the merge request will need to become full qualified URLS.  Looking back at my message, I notice I am not clear and missed specificying that all of the changes need to  updated to fully qualified URLS.  

Ex.  `- label: Follow our [Code of Conduct](CODE_OF_CONDUCT.md)` to become
`- label: Follow our [Code of Conduct](https://github.com/uab-cgds-worthey/rosalution/blob/main/CODE_OF_CONDUCT.md)`

Reply:
Thank you for the clarity. I believe I have now made these requested changes.